lib/libutee/include/tee_api.h

```diff
@@ -114,7 +114,7 @@ bool TEE_MaskCancellation(void);
 TEE_Result TEE_CheckMemoryAccessRights(uint32_t accessFlags, void *buffer,
 				       uint32_t size);
 
-void TEE_SetInstanceData(void *instanceData);
+void TEE_SetInstanceData(const void *instanceData);
```

Comment:
most user (at least in the xtest and the GP suite) do cast the return value of `TEE_GetInstanceData()`, hence easily discarding `const` when required. Why not comply with the spec?